src/concurrency/init_once.rs

```diff
@@ -2,7 +2,7 @@ use std::collections::VecDeque;
 
 use rustc_index::Idx;
 
-use super::thread::DynUnblockCallback;
+use super::thread::DyMachineCallback;
```

Comment:
You have changes in the first commit which are undone by the second commit -- please clea that up by squashing the two commits.